gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -178,12 +178,8 @@ def create_postgres_root_user
     end
 
     def create_postgres_database
-      run_as_postgres("createdb -E utf8 -O #{username} #{database}")
-    end
-
-    # some overlap with PostgresAdmin
-    def run_as_postgres(cmd)
-      AwesomeSpawn.run("su", :params => {"-" => nil, nil => "postgres", "-c" => "#{PostgresAdmin.scl_enable_prefix} \"#{cmd}\""})
+      conn = PG.connect(:user => "postgres", :dbname => "postgres")
+      conn.exec("CREATE DATABASE #{database} OWNER #{username} ENCODING 'utf8'")
```

Comment:
If either command (`PG.connect`, or `conn.exec`) fails a PG::Error will be raised.  In the previous implementation the error would be returned in the `AwesomeSpawn::CommandResult`, but that is not checked anywhere so would fail silently.  By raising an exception we will rescue and log the error here https://github.com/ManageIQ/manageiq/blob/master/gems/pending/appliance_console/logging.rb#L80